util/util.go

```diff
@@ -37,6 +37,15 @@ var (
 	}
 )
 
+// OutputConfig represents values to be passed to
+// the OutputResults function.
+type OutputConfig struct {
+	JSONFmt           bool
```

Comment:
This would be fine to start with, but I think it would be better to have an `Outputter` interface because there might be more options in future. In fact kube-bench already has regular output, JSON output, and an option to write output to a [postgres database](https://github.com/aquasecurity/kube-bench/blob/787bf6ca4d19ac7d554b0c273099137a836bb493/cmd/common.go#L121). Wdyt?

Reply:
hmm... that's a great idea. I can definitely see the need to send the output to NoSQL dbs and the interface would make it easier to add different implementations. One thing I'm of is how to strike a balance between bench-common providing default `Outputters` while allowing *-bench specific extensions.